### DISCUSSION TITLE
How to create monitored item?

### DISCUSSION THREAD

**Discussion by TomasRychy** [FreeOpcUa/opcua-asyncio#1450] [Q&A]
Hello,

I need a help because I have a problem with creation of monitored items.
My task is create the monitored item for some node.

I found the method `create_monitored_items` under subscription object, but I'm not sure how to use it.

Can you help me with that?

My task is create only the monitored item with status code Good.

Thank you and best regards

**Comment by AndreasHeine** [comment]
https://github.com/AndreasHeine/opcua-tutorial/blob/main/client/subscribe.py

**Reply by TomasRychy** [reply]
Thank you, I think that I can use whole subscription for it. I will test both at the same time.
I will test the subscription and creation of monitored items in the same time.
For the subscription you need to create monitored item and if subscription works then the monitored item was also created well.
